### PR TITLE
structed Logs

### PR DESCRIPTION
**Pull request by wyyxd2017**
what I submmit
[Introducing Structured Logs](https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/)，replaced the Klog.infof to Klog.infoS

### CODE REVIEW

**Comment by xunpan** [comment]
@wyyxd2017 
Thanks for your trying to make `kubefed` with updated log functions. 
Please:
1. make a common rule in kubefed to use for the new log format. To me, this is not mandatory change but you can show your rule out to let people review more details.
2. please address all related part instead of one by one
3. please avoid duplicate PRs

File: cmd/controller-manager/app/controller-manager.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func Run(opts *options.Options, stopChan <-chan struct{}) error {
 
 	if opts.Scope == apiextv1.NamespaceScoped {
 		opts.Config.TargetNamespace = opts.Config.KubeFedNamespace
-		klog.Infof("KubeFed will be limited to the %q namespace", opts.Config.KubeFedNamespace)
+		klog.InfoS("KubeFed will be limited to the namespace", opts.Config.KubeFedNamespace)
```
Comment (makkes) [inline cmd/controller-manager/app/controller-manager.go:152]: this is missing a key:
```suggestion
		klog.InfoS("KubeFed will be limited to the namespace", "namespace", opts.Config.KubeFedNamespace)
```

File: cmd/controller-manager/app/controller-manager.go [hunk 2]
```diff
@@ -239,14 +239,14 @@ func getKubeFedConfig(opts *options.Options) *corev1b1.KubeFedConfig {
 
 		err := client.Get(context.Background(), fedConfig, namespace, name)
 		if apierrors.IsNotFound(err) {
-			klog.Infof("Cannot retrieve KubeFedConfig %q: %v. Default options will be used.", qualifiedName.String(), err)
+			klog.InfoS("Cannot retrieve KubeFedConfig", "KubeFedConfig", qualifiedName.String(), "err", err,". Default options will be used.")
```
Comment (makkes) [inline cmd/controller-manager/app/controller-manager.go:242]: I wonder why we're not logging at error level here like `klog.ErrorS`. 🤔

**Comment by makkes** [comment]
@wyyxd2017 Currently kubefed is using klog v1.0.0 but in order ot make use of the structured logging functions, we'd need to bump to v2: https://pkg.go.dev/k8s.io/klog/v2

**Comment by hectorj2f** [comment]
@wyyxd2017 Did you have time to address these comments ?

**Pull request closed without merging**